pkg/access_token/access_token.go

```diff
@@ -0,0 +1,15 @@
+package access_token
+
+/*
+	This interface handles a Conjur access token. Structs implementing this interface should have the ability to read,
```

Comment:
Let me know what you think of this rephrasing. The intention was to:
1. highlight that AccessToken represents an access token.
2. elaborate on the example a bit
3. strip the first person plural to shift the focus of actor to conjur-k8s-secrets-manager


```
This interface represents a Conjur access token residing in some arbitrary storage medium (e.g. in-app memory or file system). Structs implementing this interface should have the ability to read,
write & delete the access token contents.
For example, conjur-k8s-secrets-manager uses memory.AccessToken. The authenticator
object is created with an empty access token which is populated with the Write method when access token contents become available. In this case the Write method writes the data to in-app memory. Later, conjur-k8s-secrets-manager can call the Read() method to access the
token-contents for retrieving secrets from Conjur.
```